core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/upgrade-master0.yml

```diff
@@ -100,7 +100,7 @@
 - name: k8s/master0 | Customize CoreDNS for latest Kubernetes ({{ version }})
   include_tasks: upgrade-coredns.yml
   when:
-    - upgrade_to_final_version
+    - version is version('1.17.4', '>=')
```

Comment:
Please add a comment next to this condition for what it is. I would also update the task name since after adding the condition it's not only for latest K8s now. Proposal: `k8s/master0 | Customize CoreDNS` # adds 'hosts' plugin

Reply:
Sure, I can add it, it will be more readable :) Thanks!:)